lib/puppet/thread_local.rb

```diff
@@ -3,5 +3,5 @@
 # We want to use the pure Ruby implementation even on JRuby. If we use the Java
 # implementation of ThreadLocal, we end up leaking references to JRuby instances
 # and preventing them from being garbage collected.
```

Comment:
nit for future us, maybe update this text as we're now calling the JRuby implmentation on JRuby, not the Ruby one?